lua/nightfox/lib/native_bitops.lua

```diff
@@ -0,0 +1,26 @@
+local function round(x)
+  return x >= 0 and math.floor(x + 0.5) or math.ceil(x - 0.5)
```

Comment:
Yes you are right I can just port them to use the lua builtin language. Will remove.

Reply:
I guess we need to `round()` as the following errors.

```
lightline.vim: Could not load colorscheme nightfox.
...ack/opt/nightfox.nvim/lua/nightfox/lib/native_bitops.lua:3: number (local 'a') has no integer representation  
```

Reply:
hmmm would be interested to know what that number is that is being left shifted.

Reply:
Looking at it further it looks like this is because in `colors.lua` the rgb values are stored as a float from 0-1. In the `to_hex` function this is still a float and needs to be changed to an int.